server/apps/immich/src/api-v1/asset/asset.core.ts

```diff
@@ -19,24 +18,23 @@ export class AssetCore {
     livePhotoAssetId?: string,
   ): Promise<AssetEntity> {
     let asset = await this.repository.create({
-      userId: authUser.id,
+      owner: { id: authUser.id } as UserEntity,
 
       mimeType: file.mimeType,
       checksum: file.checksum || null,
       originalPath: file.originalPath,
 
-      createdAt: timeUtils.checkValidTimestamp(dto.createdAt) ? dto.createdAt : new Date().toISOString(),
-      modifiedAt: timeUtils.checkValidTimestamp(dto.modifiedAt) ? dto.modifiedAt : new Date().toISOString(),
-      updatedAt: new Date().toISOString(),
-
       deviceAssetId: dto.deviceAssetId,
       deviceId: dto.deviceId,
 
+      fileCreatedAt: dto.createdAt,
+      fileModifiedAt: dto.modifiedAt,
+
       type: dto.assetType,
       isFavorite: dto.isFavorite,
       duration: dto.duration || null,
       isVisible: dto.isVisible ?? true,
-      livePhotoVideoId: livePhotoAssetId || null,
+      livePhotoVideo: livePhotoAssetId != null ? ({ id: livePhotoAssetId } as AssetEntity) : null,
```

Comment:
Why this change here? We still have the `livePhotoVideoId` field, right?

Reply:
We do, but I think we can remove those in general and use this way of writing existing assets.